package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "j5-songs",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
Please don't update package.json since I do this on a different commit, thanks :)